sdk/storage/azure-storage-common/src/main/java/com/azure/storage/common/implementation/PayloadSizeGate.java

```diff
@@ -38,18 +38,12 @@
      * @return Buffered data or incoming data depending on threshold condition.
      */
     Flux<ByteBuffer> write(ByteBuffer buf) {
-        /* TODO (gapra): Investigate if there is way to avoid copying the data twice since this may result in lower
```

Comment:
Should we add a changelog entry for common, though all these changes are "internal only" changes